test/presubmit-tests.sh

```diff
@@ -23,26 +23,9 @@
 
 source $(dirname $0)/../vendor/github.com/knative/test-infra/scripts/presubmit-tests.sh
 
-function build_tests() {
-  header "Verifying code structure and contents"
+# We use the default build test runner.
```

Comment:
This line is duplicated (mostly)

Reply:
No, it's not. Each one refers to a different runner: build, unit and integration.